applications/main/momentum_app/scenes/momentum_app_scene_interface_general.c

```diff
@@ -0,0 +1,61 @@
+#include "../momentum_app.h"
+
+enum VarItemListIndex {
+    VarItemListIndexMidnightFormat,
+};
+
+void momentum_app_scene_interface_general_var_item_list_callback(void* context, uint32_t index) {
+    MomentumApp* app = context;
+    view_dispatcher_send_custom_event(app->view_dispatcher, index);
+}
+
+static void momentum_app_scene_interface_general_midnight_format_changed(VariableItem* item) {
+    MomentumApp* app = variable_item_get_context(item);
+    bool value = variable_item_get_current_value_index(item);
+    variable_item_set_current_value_text(item, value ? "00:XX" : "XX:00");
```

Comment:
im assuming this is supposed to be
```suggestion
    variable_item_set_current_value_text(item, value ? "00:XX" : "12:00");
```